flink/v1.19/flink/src/test/java/org/apache/iceberg/flink/sink/TestFlinkIcebergSinkV2.java

```diff
@@ -140,8 +140,8 @@ public void testCheckAndGetEqualityFieldIds() {
 
     DataStream<Row> dataStream =
         env.addSource(new BoundedTestSource<>(ImmutableList.of()), ROW_TYPE_INFO);
-    FlinkSink.Builder builder =
-        FlinkSink.forRow(dataStream, SimpleDataUtil.FLINK_SCHEMA).table(table);
+    IcebergSink.Builder builder =
```

Comment:
What was the original intention of this test? Why it was V2? Do we still has test for that case?
Same question for `TestFlinkIcebergSinkV2Base`

Reply:
There was already a unit test file called `TestFlinkIcebergSink`. So I cloned its changed it to use `IcebergSink` and rename the file accordingly. This same applies to `TestFlinkIcebergSinkV2Base`